Engine/Main/GameObject.h

```diff
@@ -90,13 +90,14 @@ class GameObject
 	void RemoveComponentsCopying(const GameObject& gameobject_to_copy);
 
 public:
-	std::vector<Component*> components;
 	std::string name = "";
-
+	uint64_t UUID = -1;
+	std::string tag = ""; // Please don't modify this parameter in a script, always use the editor!
```

Comment:
Maybe it's better to put it private and have a Getter in order to avoid anyone outside the editor modify the tag.

Reply:
It could, but there are some places in the engine where it has to be modified. Due to avoiding putting friends everywhere I prefered setting it as a public